Documentation for time-series geo_line

craigtaverner opened the pull request:
The feature introduced in https://github.com/elastic/elasticsearch/pull/94954 did not include documentation, so here it is!

craigtaverner committed:
Documentation for time-series geo_line

elasticsearchmachine commented:
Pinging @elastic/es-analytics-geo (Team:Analytics)

elasticsearchmachine commented:
Pinging @elastic/es-docs (Team:Docs)

craigtaverner committed:
Fix incorrect ids in geoline docs

iverase reviewed (approved):
LGTM, reads nice, thanks

craigtaverner committed:
Some updates from review

Added image of kibana map, improved first example, linked to TSDS and added section on line simplification with link to wikipedia.

craigtaverner committed:
Diagrams of truncation versus simplification
